apitools/base/protorpclite/messages.py

```diff
@@ -758,6 +758,8 @@ class Order(Message):
 
     """
 
+    __hash__ = object.__hash__
```

Comment:
I think hash should be consistent with implementation of `__eq__`, so that equal messages have equal hashes.